site/app/libraries/Output.php

```diff
@@ -514,8 +518,12 @@ public function addCss(string $url): void {
         $this->css->add($url);
     }
 
+    public function addInternalModuleTwigJs(string $file) {
+        $this->addModuleJs($this->timestampResource($file, 'mjs/twig'));
+    }
+
     public function addInternalModuleJs(string $file) {
-        $this->addModuleJs($this->timestampResource($file, 'mjs'));
+        $this->addModuleJs($this->timestampResource($file, 'mjs/module'));
```

Comment:
Is there a reason to move the existing js and ts files into a module directory under `ts`? Those files were intended to be the top level entry points and any files in a subdirectory would be bundled into those entry points.

The `mjs` typically indicates ecma script module so I think the second dir is redundant